src/com/mambu/apisdk/services/SavingsService.java

```diff
@@ -254,6 +255,28 @@ public SavingsAccount undoApproveSavingsAccount(String accountId, String notes)
 		return serviceExecutor.execute(getAllSavingsTransactions, params);
 	}
 
+	/**
+	 * Get savings transactions by specifying filter constraints
+	 * 
+	 * @param filterConstraints
+	 *            filter constraints. Must not be null
+	 * @return list of savings transactions matching filter constraints
+	 * @throws MambuApiException
+	 */
+	public List<SavingsTransaction> getSavingsTransactions(JSONFilterConstraints filterConstraints, String offset,
+			String limit) throws MambuApiException {
+		// Available since Mambu 3.12. See MBU-8988 for more details
+		// POST {JSONFilterConstraints} /api/savings/transactions/search?offset=0&limit=5
+
+		ApiDefinition apiDefintition = SearchService
+				.makeApiDefinitionforSearchByFilter(MambuEntityType.LOAN_TRANSACTION);

```

Comment:
Good catch. It's a bug and it needs to be changed to SAVINGS_TRANSACTION instead. Thanks!